src/test/java/tests/TestBuyInSearchShoes.java

```diff
@@ -0,0 +1,79 @@
+package tests;
+
+import base.AbstractBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.PageBuyInSearchShoes;
+
+import static java.lang.Thread.sleep;
+
+public class TestBuyInSearchShoes extends AbstractBaseTest {
+    @Test
+    public void testBuyInSearchShoes() throws InterruptedException {
+        HomePage homePage = new HomePage(driver);
+        PageBuyInSearchShoes pageBuyInSearchShoes = new PageBuyInSearchShoes(driver);
+
+        homePage.closePopUp()
+                .closeCookie()
+                .closeCookieOff()
+                .closeCookieEmail()
+                .clickSearchButton();
+
+        sleep(2000);
+        pageBuyInSearchShoes
+                .clickBmwShoes()
+                .closeCookieInBmwPage()
+                .clickGenderButton()
+                .clickMenButton();
+        int menCategoryNumberOfItem = pageBuyInSearchShoes.getTextGenderMenCategories();
+        pageBuyInSearchShoes.closeFilter();
+
+        pageBuyInSearchShoes.getElementsCountByLocatorMen();
+
+        Assert.assertEquals(menCategoryNumberOfItem, pageBuyInSearchShoes.getElementsCountByLocatorMen());
+    }
+
+    @Test
+    public void testPumaPage() {
```

Comment:
why is this test in the same class as the previous test?

Reply:
I wrote separately to shorten the test + there is a separate thread per page